apps/bus_detective_web/assets/js/stop-map.js

```diff
@@ -29,6 +29,13 @@ class StopMap extends HTMLElement {
     map.addLayer(Leaflet.tileLayer(TILE_URL, {detectRetina: true}));
     Leaflet.layerGroup().addTo(map);
     Leaflet.marker(center).addTo(map);
+
+    let toggleButton = document.querySelector('#toggleButton');
+
+    toggleButton.addEventListener('click', function (event) {
+      document.querySelector('#stopMap').classList.toggle('map--expanded');
+      map.invalidateSize();
+    });
```

Comment:
Having the stop map listen to the toggle button which is in another component feels not quite right. We are now getting to the point where we have 2 components that need to communicate, and there's nothing in plain ole web components to help us there. I'll try to spike something out using custom events and send a separate PR.